packages/ui/components/form/color-picker/colorpicker.tsx

```diff
@@ -33,7 +33,7 @@ const ColorPicker = (props: ColorPickerProps) => {
           <Popover.Content align={props.popoverAlign ?? "center"} sideOffset={10}>
             <HexColorPicker
               color={color}
-              className="!h-32 !w-32"
+              className="pointer-events-auto !h-32 !w-32"
```

Comment:
Why do we need this?

Reply:
We needed it for the color picker to work before but it does not seem to be needed now. Will revert the change